lib/routes.js

```diff
@@ -35,8 +36,9 @@ const statsClient = new StatsClient(localCacheClient, STATS_INTERVAL,
     STATS_EXPIRY);
 
 function checkUnsuportedRoutes(req, res, log) {
-    if (req.query.policy !== undefined ||
-        req.query.tagging !== undefined) {
+    const unsupportedRequestQuery = Object.keys(req.query)
```

Comment:
do you think it makes sense to add this commit to avoid parsing the url again here, too?: 
https://github.com/scality/S3/pull/603/commits/87d91876ccb8809e08faa8cb519ee29df9b5a046